frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.html

```diff
@@ -1,72 +1,187 @@
                     >
                     </app-transfer-point-name>
+                    <ng-template #noTargetTransferPoint>
+                        <span class="text-muted"> Kein Ziel ausgewählt </span>
+                    </ng-template>
                 </button>
+                <div ngbDropdownMenu>
+                    <button
+                        ngbDropdownItem
+                        (click)="
+                            firstVehiclesToOtherLocationTransferPointId =
+                                undefined
+                        "
+                        type="button"
+                        class="btn btn-primary"
+                        *ngIf="
+                            firstVehiclesToOtherLocationTransferPointId !==
+                            undefined
+                        "
+                    >
+                        Kein Ziel
+                    </button>
+                    <ng-container
+                        *ngFor="
+                            let transferPoint of transferPoints;
+                            trackBy: 'id' | appTrackByProperty : transferPoints
+                        "
+                    >
+                        <button
+                            ngbDropdownItem
+                            (click)="
+                                firstVehiclesToOtherLocationTransferPointId =
+                                    transferPoint.id
+                            "
+                            type="button"
+                            class="btn btn-primary"
+                            *ngIf="
+                                targetTransferPointId !== transferPoint.id &&
+                                firstVehiclesToOtherLocationTransferPointId !==
+                                    transferPoint.id
+                            "
+                        >
+                            <app-transfer-point-name
+                                [transferPointId]="transferPoint.id"
+                            >
+                            </app-transfer-point-name>
+                        </button>
+                    </ng-container>
+                </div>
             </div>
         </div>
-    </h5>
 
-    <div *ngIf="alarmGroups$ | async | values as alarmGroups" class="card-body">
-        <div *ngIf="!targetTransferPointId" class="text-danger mb-3">
-            Bitte wählen Sie zuerst ein Ziel aus.
+        <div class="mt-2">
+            <div *ngIf="!targetTransferPointId" class="text-danger">
+                Bitte wählen Sie zuerst ein Ziel aus.
+            </div>
+            <div
+                *ngIf="
+                    targetTransferPointId &&
+                    !firstVehiclesToOtherLocationTransferPointId &&
+                    firstVehiclesToOtherLocationCount > 0
+                "
+                class="text-warning"
+            >
+                Ohne ein zweites Ziel können die ersten Fahrzeuge kein anderes
+                Ziel anfahren.
```

Comment:
Either make it more explicit that the vehicles will head to the default destination in this case or disable the buttons to send an alarm group if `!firstVehiclesToOtherLocationTransferPointId && firstVehiclesToOtherLocationCount > 0`. At least for me it was not obvious what will happen if I send an alarm group in this case.

Reply:
Done in d81b5f0823d86c3041b6c8f305072b1fcd177014